messages/snapshot.delete.md

```diff
@@ -4,7 +4,7 @@ Delete a scratch org snapshot.
 
 # description
 
-Dev Hub admins can delete any snapshot, while users can delete only their own unless a Dev Hub admin gives the user Modify All permissions.
+Dev Hub admins can delete any snapshot, while users can delete only their own unless a Dev Hub admin gives the user Modify All permissions (Salesforce license only).
```

Comment:
```suggestion
Dev Hub admins can delete any snapshot. Users can delete only their own snapshots, unless a Dev Hub admin gives the user Modify All permission, which works only with the Salesforce license.
```

Reply:
@agomez-sf , I checked with EmilyK about this.  I felt like it was unclear what exactly the _Salesforce license only_ part was qualifying. Thank you!

Reply:
Sounds good, I've updated the description.Thank you!